test/integration/targets/aws_codebuild/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+unsupported
```

Comment:
You are turning the integration tests off.
So, the integration tests you wrote are not executed.